desktop/apps/auction2/components/filter_sort/index.js

```diff
@@ -10,6 +10,10 @@ function FilterSort(props) {
     updateSortAction
   } = props
   const selectedSort = filterParams.sort
+  const itemHeight = 37
```

Comment:
Hm, yeah, not sure which is less evil. Is there no react-y tools for introspecting element size? It's not the worst thing, would just be annoying to change whenever the styling changes.